GitHub-trainings/github-pr-training/latex-slides/GitHub-PR-training.tex

```diff
@@ -0,0 +1,806 @@
+		\vspace{.7cm}\newline
+		\large We will go over all steps but focus today on the \textbf{work stage}	
+
+		\column{.65\textwidth} % Right column and width
+		\vspace{-.75cm}
+		\begin{figure}
+			\centering
+			\includegraphics[width=\textwidth]{./img/branch-pr-merge-cycle.png}
+		\end{figure}
+		
+	\end{columns}
+\end{frame}
+
+\begin{frame}
+	\frametitle{Stage: The set up stage}
+	
+	\huge\centering \textbf{SET UP STAGE}
+	
+\end{frame}
+
+
+\begin{frame}
+	\frametitle{Stage: The set up stage}
+	\begin{columns}[c]
+
+		\column{.35\textwidth} % Left column and width
+		
+		\Large \textbf{The set up stage:}
+		\vspace{1em}
+		\normalsize
+		
+		\begin{itemize}
+			\setlength\itemsep{1em}
+			\item Document the task you will work on
+			\item Create a space in your repo to work on the task
+		\end{itemize}
+		
+		\column{.65\textwidth} % Right column and width
+		\vspace{-.75cm}
+		\begin{figure}
+			\centering
+			\includegraphics[width=\textwidth]{./img/branch-pr-merge-cycle-S1.png}
+		\end{figure}
+		
+	\end{columns}
+\end{frame}
+
+
+\begin{frame}
+	\frametitle{Step: Create an issue}
+	\begin{columns}[c]
+		
+		\column{.35\textwidth} % Left column and width
+		\begin{itemize}
+			\setlength\itemsep{.5em}
+			\item Issues are a great way to document what tasks need to be done
+			\item They allow other team members to provide input on how to solve the task
+			\item No need to open an issue for tasks that will be done immediately and don't require a place to document related discussions
+		\end{itemize}
+		
+		\column{.65\textwidth} % Right column and width
+		\vspace{-.75cm}
+		\begin{figure}
+			\centering
+			\includegraphics[width=\textwidth]{./img/branch-pr-merge-cycle-S1-1.png}
+		\end{figure}
+		
+	\end{columns}
+\end{frame}
+
+\begin{frame}
+	\frametitle{Create an issue}
+	\begin{columns}[c]
+
+		\column{.55\textwidth} % Right column and width
+		\vspace{-.5cm}
+		\begin{figure}
+			\centering
+			\includegraphics[width=\textwidth]{./img/create-issue-1.png}
+		\end{figure}
+		
+		\column{.45\textwidth} % Left column and width
+		
+		\begin{itemize}
+			\setlength\itemsep{.5em}
+			\item Click the issues tab and then \texttt{New Issue}
+			\item Document the task and/or ask the rest of the team of input
+			\item Assign yourself or someone else to complete the task
+			\item Add labels to facilitate future searches
+			\item This is a far superior way to document tasks compared to emails or in-person meetings!
+		\end{itemize}
+			
+	\end{columns}	
+\end{frame}
+
+\begin{frame}
+	\frametitle{Browse an issue}
+	\begin{columns}[c]
+		
+		\column{.55\textwidth} % Right column and width
+		\vspace{-.5cm}
+		\begin{figure}
+			\centering
+			\includegraphics[width=\textwidth]{./img/create-issue-2.png}
+		\end{figure}
+		
+		\column{.45\textwidth} % Left column and width
+		
+		\begin{itemize}
+			\setlength\itemsep{1em}
+			\item Take note of the issue number (also found in URL) so you can refer to it
```

Comment:
not super important in my opinion, but perhaps it would be useful to include an example URL so the slide shows where exactly is the issue number

Reply:
Done in f5ceeb42a1e713bce0567b068e47351742f6447b - I did not add a full URL as it took up too much space, but I indicated that the issue number is the number in the end of the URL